site/src/components/UserDropdownContent/UserDropdownContent.tsx

```diff
@@ -73,12 +93,49 @@ export const UserDropdownContent: FC<UserDropdownContentProps> = ({
         </MenuItem>
       </a>
 
-      <MenuItem className={styles.menuItem} onClick={onSignOut}>
-        <ListItemIcon className={styles.icon}>
-          <LogoutIcon />
-        </ListItemIcon>
-        <ListItemText primary={Language.signOutLabel} />
-      </MenuItem>
+      <a
+        href={githubUrl}
+        target="_blank"
+        rel="noreferrer"
+        className={styles.link}
+      >
+        <MenuItem className={styles.menuItem} onClick={onPopoverClose}>
+          <ListItemIcon className={styles.icon}>
+            <BugIcon />
+          </ListItemIcon>
+          <ListItemText primary={Language.bugLabel} />
+        </MenuItem>
+      </a>
+
+      <a
+        href={discordUrl}
+        target="_blank"
+        rel="noreferrer"
+        className={styles.link}
+        title="View Source"
+      >
+        <MenuItem className={styles.menuItem} onClick={onPopoverClose}>
+          <ListItemIcon className={styles.icon}>
+            <ChatIcon />
+          </ListItemIcon>
+          <ListItemText primary={Language.discordLabel} />
+        </MenuItem>
+      </a>
+
+      <Divider />
+
+      <Tooltip title="Browse Source Code">
```

Comment:
Just curious why a `Tooltip` over the native `title` prop?  So it pops up faster maybe?

Reply:
The `title` property wasn't appearing for me for some reason...

Reply:
Ahh gotcha, there was a `title="View Source"` on the Discord link so it might have been on the wrong one!

Reply:
Ahh my bad. I'll fix!